### DISCUSSION TITLE
How to use prism.js plugin

### DISCUSSION THREAD

**Discussion by officialrajdeepsingh** [lumeland/lume#326] [Q&A] [answered]
I need some additional[ prism.js plugins](https://prismjs.com/#plugins) for my blog. But I can't find a way to install it.

**Comment by oscarotero** [comment, marked as answer]
Prism is the only Lume dependency loaded from `esm.sh` instead of `npm:`. This makes difficult to install plugins ATM. The reason for using `esm.sh` is it fails on `npm:` on loading languages dynamically (see https://github.com/denoland/deno/issues/15826).

I need to make some breaking changes in the plugin to switch to npm. Probably in 1.14.0.
